### ISSUE TITLE
Compare the patches to AM2_CPU by CFOU! to the ones by Pyrdacor

### ISSUE THREAD

**Issue by a1exh** [Pyrdacor/Ambermoon#93]
@dlfrsilver kindly provided the source code to the resourced AM2_CPU by CFOU! which included numerous fixes they did while developing the French version.

Whilst eyeballing the commented changes I could see one or two that didn't seem to have an equivalent in the work @Pyrdacor has done.

The documented changes to AM2_CPU by @Pyrdacor include :

- eagle memory crash
- remove curse spell
- SP stealer
- diseases
- rune table
- skill penalties for items
- spelling mistakes
- item values

I think it would be interesting to try to understand all the fixes that CFOU! and dlfrsilver did to ensure nothing has been missed.

**Comment by Pyrdacor** [comment]
Good point. For 1.19 I plan to drop the Ghidra compatibility which means that I will directly modify the exported AM2_CPU assembler file instead of trying to add code changes in Ghidra.

All open code fixes should be possible to fix then much more easily.

I will also try to implement the mentioned things as well.

**Comment by Pyrdacor** [comment]
Did you find some specific code fixes?

**Comment by a1exh** [comment]
DIFF the files "am2_cpu_final.asmgood5" against "am2_cpu_final.asm"

The first file I believe is am2_cpu for v1.07 resourced but not localised (possibly already with some bugfixes). 
The 2nd is the final version they did for their Ambermoon French.

Line 7150 is the fix for remove curse. 
Line 26171 is ???
Line 33292 is ???
Line 34240 is ???
Line 34664 is the fix for SP stealer

Plus there are a lot of comments at the start of the file.

Maybe they are nothing? Bugs in their re-sourcing of the binary?
